pkg/grpcclient/client.go

```diff
@@ -53,6 +53,15 @@ func SetupDomainsClient(ctx context.Context, cfg Config) (grpcDomainsV1.DomainsS
 		return nil, nil, err
 	}
 
```

Comment:
I have run them and adding health check does cause an issue as they are codependent

Reply:
Yes, Because they all have cyclic dependencies, 
so we can remove the health check for following  ?
- `SetupDomainsClient`
- `SetupClientsClient`
- `SetupChannelsClient`
- `SetupGroupsClient`